src/wcl/filepath.h

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
This is part of another PR but I rebased it in slightly differently so the PR diff isn't correct. I'll land the other thing soon and rebase this to fix it. Just ignore this for now.